src/eventProcessor.ts

```diff
@@ -193,15 +213,103 @@ export default async (event: ContractEvent): Promise<void> => {
       return;
     }
 
+    case ContractEventsSignatures.ExtensionAddedToNetwork: {
+      const { extensionId, version } = event.args;
+
+      await mutate('setCurrentVersion', {
+        input: {
+          item: extensionId,
+          version: BigNumber.from(version).toNumber(),
+        },
+      });
+
+      return;
+    }
+
+    case ContractEventsSignatures.ExtensionInstalled: {
```

Comment:
While not really necessary, I think it would be useful to have at least some basic `verbose()` calls when handling each event.

Ie: something like `Extension <extensionAddress> has been installed in colony <colonyAddress>`

Especially since `verbose` calls will not be visible in production

Reply:
Agree